src/main/scala/treadle/TreadleOptions.scala

```diff
@@ -0,0 +1,282 @@
+// See LICENSE for license details.
+
+package treadle
+
+import firrtl.CircuitState
+import firrtl.annotations.{Annotation, NoTargetAnnotation}
+import firrtl.ir.Circuit
+import firrtl.options.{HasShellOptions, ShellOption, Unserializable}
+import firrtl.stage.FirrtlSourceAnnotation
+import treadle.executable.{ClockInfo, TreadleException}
+
+sealed trait TreadleOption extends Unserializable { this: Annotation => }
+
+/**
+  * Tells treadle to write a vcd file during simulation
+  */
+case object WriteVcdAnnotation extends NoTargetAnnotation with TreadleOption with HasShellOptions {
+  val options: Seq[ShellOption[_]] = Seq(
+    new ShellOption[Unit](
+      longOption = "tr-write-vcd",
+      shortOption = Some("tiwv"),
```

Comment:
I'm not sure what that alternative is here. I'd love to get rid of the alphabet soup. Are you suggesting a set prefix for each CLI or dropping short forms altogether

Reply:
Dropping short forms altogether.

This is still backwards compatible as you have the old driver/execution options if (for whatever reason) you really want to use the old short options. We'd then break backwards compatibility by removing the drivers in the x.3 major release. For a stage/phase port, because backwards compatibility is preserved in the old driver, you have a free hand here. Don't feel like you're backed into a corner with the old short options. As far as I'm concerned, you have a free hand to define these however you want. If you're cool with it as is, then leave it.

I do agree that there should be a way for a stage to define a prefix and all options get that. That would have to start as a FIRRTL PR and percolate out, though.